src/main/java/com/yelp/nrtsearch/server/config/ThreadPoolConfiguration.java

```diff
@@ -39,6 +43,8 @@ public class ThreadPoolConfiguration {
   private static final int DEFAULT_MAX_GRPC_REPLICATIONSERVER_BUFFERED_ITEMS =
       DEFAULT_MAX_INDEXING_BUFFERED_ITEMS;
 
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
Can we use this ObjectMapper? https://github.com/Yelp/nrtsearch/blob/master/src/main/java/com/yelp/nrtsearch/server/utils/JsonUtils.java